package.json

```diff
@@ -42,7 +42,7 @@
     ]
   },
   "dependencies": {
-    "@bugsnag/react-native": "^7.3.2",
+    "@bugsnag/react-native": "^7.3.4",
```

Comment:
Previous versions of Bugsnag have some issues with Proguard, so I had to update to the latest one.
https://github.com/bugsnag/bugsnag-js/blob/next/CHANGELOG.md